sql/gen-sql-config-docs.py

```diff
@@ -119,17 +119,13 @@ def generate_sql_configs_table_html(sql_configs, path):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 2:
-        print("Usage: ./bin/spark-submit sql/gen-sql-config-docs.py <static|runtime>")
-        sys.exit(-1)
-    else:
-        group = sys.argv[1]
-
```

Comment:
Plz describe why we need this update in the PR description? Probably, this is a followup of the Hyukjin comment ?
https://github.com/apache/spark/pull/28274#discussion_r412893096